src/account_manager/operations/participation.rs

```diff
@@ -62,6 +62,20 @@ impl AccountManager {
             .collect())
     }
 
+    pub async fn get_participation_events_from_client(
```

Comment:
Please add a small documentation for the methods since it's a public one.
Also, maybe the name is confusing since we are not returning events but events ids.
I'd drop the `_client` and call it `get_participation_event_ids` ?

Reply:
I can add the IDs, but it might be worth adding a second argument to the function `nodes: Option<Vec<Node>>` then if no nodes are provided we go with the client options one.

Reply:
Even without an additional parameter I feel like it's redundant. It's quite obvious it calls the client, if we follow the logic we would need to add that prefix to a LOT of wallet methods...

Reply:
That's fair. In that case, I'll add the second optional argument.

Reply:
That's fair. I'll change it and not add the optional parameter, since it should be no problem to add later should we decide we need it :+1: